types/resource.go

```diff
@@ -285,4 +286,7 @@ type BackupStatus struct {
 type RestoreStatus struct {
 	IsRestoring  bool   `json:"isRestoring"`
 	LastRestored string `json:"lastRestored"`
+	Progress     int    `json:"progress,omitempty"`
+	Error        string `json:"restoreError,omitempty"`
```

Comment:
just `error` for the json to keep it consistent.

Reply:
I've sent an engine PR with the modifications that addresses this change